### DISCUSSION TITLE
Building Dapp to earn income during strikes

### DISCUSSION THREAD

**Discussion by Im-Masoud** [WeAreMahsaAmini/Core#267] [Ideas]
We can make a Dapp similar to forsage with a few minor changes to make money for people and help the strikers.

Forsage is Marketing Dapp that everybody can earn from it.

**Comment by ghost** [comment]
Can you explain a bit? I'm not familiar with Dapp or Forsage

**Reply by Im-Masoud** [reply]
Hi it is marketing Dapp that anyone can creat own team and earn from it.
